src/H5VLconnector.h

```diff
@@ -1105,14 +1105,6 @@ H5_DLL void *H5VLobject(hid_t obj_id);
  * \ingroup H5VLDEV
  */
 H5_DLL hid_t H5VLget_file_type(void *file_obj, hid_t connector_id, hid_t dtype_id);
-/**
- * \ingroup H5VLDEV
- */
-H5_DLL hid_t H5VLpeek_connector_id_by_name(const char *name);
-/**
- * \ingroup H5VLDEV
- */
-H5_DLL hid_t H5VLpeek_connector_id_by_value(H5VL_class_value_t value);
```

Comment:
This definitely needs a RELEASE.txt note

Reply:
Yes, will be in next PR, coming shortly.